code/datums/uplink_item.dm

```diff
@@ -991,6 +991,7 @@ var/list/uplink_items = list()
 	reference = "BRHS"
 	item = /obj/item/weapon/storage/box/syndie_kit/hardsuit
 	cost = 8
+	excludefrom = list(/datum/game_mode/nuclear)
```

Comment:
Well duh
But you know what you're signing up for a little better when you by a balloon, and everyone knows you're being that guy (at least intentionally)